armadillo1/UPGRADE.md

```diff
@@ -0,0 +1,13 @@
+# Upgrading
+Here we talk about the upgrading of the Armadillo suite and the updating of the other parts of the system.
+
+### Upgrading the Operating System(OS)
+To keep the system up to date you will need to patch and/or update the system. This upgrading is <ins>NOT</ins> part of the Armadillo suite. It is the task of the owner of the server or the IT department to keep the system up to date and apply security patches. When an application of the Armadillo suite is fulnerable then we going to make sure that there will be a patch made and distributed to the systems involved.
+
+### Upgrading the containers
```

Comment:
Please clarify what containers are kept up to date, to avoid missundestandings.

Reply:
done